src/shared/ContentsTable/TableEntries/PullEntries/PullFileEntry.jsx

```diff
@@ -6,16 +6,20 @@ import { displayTypeParameter } from '../../constants'
 import FileEntry from '../BaseEntries/FileEntry'
 
 function PullFileEntry({
-  commitSHA,
+  commitSha,
   path,
   isCriticalFile,
   name,
   urlPath,
   displayType,
+  filters,
 }) {
+  const flags = filters?.flags?.length > 0 ? filters?.flags : []
+
   const { runPrefetch } = usePrefetchPullFileEntry({
     path,
-    ref: commitSHA,
+    commitSha,
```

Comment:
I don't see the `usePrefetchPullFileEntry` being updated in the PR so you'll still need to alias `ref` to the `commitSha`

Reply:
Ah shoot good catch, not sure how that happened. Ty